lxdock/client.py

```diff
@@ -2,5 +2,4 @@
 
 
 def get_client():
-    """ Returns a PyLXD client to be used to orchestrate containers. """
```

Comment:
I'm just wondering why remove this docstring, it seems a bit unrelated?

Reply:
Typo, will fix.

Reply:
TBH I find this whole file to be somewhat redundant now that we don't need to do something special to get the client, but yeah.